src/api/hypixel.ts

```diff
@@ -33,6 +35,15 @@ const ratelimit = {
 async function hypixelRequest<T = unknown>(opts: HypixelRequestOptions = { usesApiKey: true }) {
 	if (!opts.endpoint) throw new Error('No endpoint provided!');
 	if (!opts.query) opts.query = {};
+
+	if (dev) {
+		// Use redis to cache requests in dev mode for easy reparsing of data
+		const cachedRequest = await REDIS.GET(`dev:hypixelrequest:${opts.endpoint}`);
```

Comment:
I think this should use the 
```ts
const requestUrl = `${baseURL}/${opts?.endpoint}?${new URLSearchParams(opts.query)}`;
```
on line 47 or else it could cause overrides

Reply:
Good catch, thanks